easybuild/easyblocks/v/vtune.py

```diff
@@ -50,6 +50,14 @@ def __init__(self, *args, **kwargs):
         elif loosever >= LooseVersion('2018'):
             self.subdir = 'vtune_amplifier'
 
+    def prepare_step(self, *args, **kwargs):
+        """Since 2019u3 there is no license check."""
+        if LooseVersion(self.version) >= LooseVersion('2019_update3'):
```

Comment:
Just a minor thing, you could just use an if here without the else, as in:

```
if LooseVersion(self.version) >= LooseVersion('2019_update3'):
    kwargs['requires_runtime_license'] = False
super(EB_VTune, self).prepare_step(*args, **kwargs)
```

Reply:
oki, I used same snippet in advisor so I'll change it in both PRs.

Reply:
fixed and build results with updated easyblock uploaded into [related easyconfig](https://github.com/easybuilders/easybuild-easyconfigs/pull/8139#issuecomment-486669786)